src/ui-kit/organisms/OnlineUsersList/OnlineUsersList.styled.ts

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import {rgba} from 'polished';
+
+export const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+`;
+
+export const List = styled.div`
+  height: 100%;
+  overflow-y: scroll;
+
+  ::-webkit-scrollbar {
+    width: 5px;
+    height: 0;
+  }
+  ::-webkit-scrollbar-thumb {
```

Comment:
make a space between selectors
```suggestion
  }
  
  ::-webkit-scrollbar-thumb {
```